desktop/cmp/panel/PanelModel.ts

```diff
@@ -157,8 +160,13 @@ export class PanelModel extends HoistModel {
         return !!this.modalSupportModel;
     }
 
+    /** True when both collapsed and not currently in a modal - i.e. *really* collapsed. */
+    get isRenderedCollapsed(): boolean {
+        return this.collapsed && !this.isModal;
+    }
```

Comment:
Struggled with this name a bit (and the newly qualified comment on `collapsed` above)...  I'm OK with where it landed but happy to discuss any alternatives.

Reply:
This distinction and naming makes sense to me 👍

Reply:
Can we add (not modal) after "home" in the description of collapsed? Might help devs get the idea faster.